README.md

```diff
@@ -36,6 +39,9 @@ The following workflows are available:
 | pre-run-script | string | "" | Path to the bash script to be run before the integration tests |
 | provider | string | microk8s | Actions operator provider as defined [here](https://github.com/charmed-kubernetes/actions-operator#usage) |
 | microk8s-addons | string | "dns ingress rbac storage" | Microk8s provider add-ons override. A minimum set of addons (the defaults) must be enabled. |
+| rockcraft-channel        | string | latest/stable | Rockcraft channel to use for the integration test |
+| rockcraft-ref            | string | "" | Used in conjunction with rockcraft-repository to pull and build rockcraft from source instead of using snapstore version. |
+| rockcraft-repository     | string | "" | Pull and build rockcraft from source instead of using snapstore version. |
```

Comment:
Would it be helpful to mention that the repository configuration overrides channel & ref? (as in it has priority over those)